scripts/explicit_kwargs_checker_tests.py

```diff
@@ -0,0 +1,56 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import astroid
+import explicit_kwargs_checker  # pylint: disable=relative-import
+
+import pylint.testutils
+
+
+class ExplicitKwargsCheckerTests(pylint.testutils.CheckerTestCase):
+    CHECKER_CLASS = explicit_kwargs_checker.ExplicitKwargsChecker
+
+    def finds_non_explicit_kwargs_test(self):
+        func_node = astroid.extract_node("""
+        def test(test_var_one, test_var_two=4, test_var_three=5, test_var_four="test_checker"): #@
```

Comment:
What's this `#@` thing?

Reply:
The `#@` symbol tells the `astroid.extract_node` to create a node for the line after which this symbol has been placed. (Here the function definition)

Reply:
Ah ok. Since other tests will probably crib from this one, maybe add a README file with generic docs like that? Or some sort of documentation somewhere where devs who want to write custom lint checks are likely to find it. (Maybe on the wiki.)

Reply:
OK, feel free to go ahead and do that now, and link to it from here! We can expand the page later but let's at least have something up.

Reply:
I have added a wiki page for [writing and testing custom checks using Pylint](https://github.com/oppia/oppia/wiki/Writing-Tests). Please see it once!